tools/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36,py37,py38,{py36,py37,py38}-flake8,{py36,py38}-mypy3
+envlist = py36,py37,py38,py39,{py36,py37,py38,py39}-flake8,{py36,py39}-mypy3
```

Comment:
```suggestion
envlist = py36,py37,py38,py39,{py36,py37,py38,py39}-flake8,{py36,py37,py38,py39}-mypy3
```

Maybe? I'm often not sure if the envlist in `tox.ini` is everything that will be tested in CI, or everything that's *possible* to test. Certainly in some cases only a subset it tested, but are there any places where we test all combinations and shouldn't sprinkle more combinations than necessary?

Reply:
In pratice I don't think we care about whether mypy in particular passes in intermediate versions of Python, but sure I guess this is reasonable.

Reply:
I pushed that change, mostly so I could test py38-mypy3 locally, since I don't have 3.9 installed.